templates/default/user/userinfobox.html

```diff
@@ -11,10 +11,21 @@
 			<IMG src="img/users.gif" alt="">
 		</TD>
 		<TD style="width: 1%;" class="bold nobr">
-			{trans("First/last name:")}
+			{trans("First name:")}
 		</TD>
 		<TD style="width: 98%;" class="fright bold">
-			{$userinfo.name}
+			{$userinfo.firstname}
```

Comment:
Jak to już przerabiasz to może od razu warto całą tabelę przerzucić na:
```html
<COLGROUP>
  ........
  <COL style="width: 1%;">
  <COL style="width: 98%;">
</COLGROUP>
```
i pozbyć się wszystkich odwołań style="width: ..." w komórkach tabel?